src/azure-cli/azure/cli/command_modules/profile/_help.py

```diff
@@ -9,42 +9,64 @@
 
 helps['login'] = """
 type: command
-short-summary: Log in to Azure.
+short-summary: Log into Azure.
 examples:
     - name: Log in interactively.
       text: >
         az login
-    - name: Log in with user name and password. This doesn't work with Microsoft accounts or accounts that have two-factor authentication enabled. Use -p=secret if the first character of the password is '-'.
+    - name: > 
+        Log in with user name and password. This doesn't work with Microsoft accounts or accounts that have two-factor authentication enabled. 
+        Use `-p=secret` if the first character of the password is `-`.
       text: >
-        az login -u johndoe@contoso.com -p VerySecret
-    - name: Log in with a service principal using client secret. Use -p=secret if the first character of the password is '-'.
+        az login --user johndoe@contoso.com --password VerySecret
+    - name: > 
+        Log in with a password containing a `-` as the first character.    
       text: >
-        az login --service-principal -u http://azure-cli-2016-08-05-14-31-15 -p VerySecret --tenant contoso.onmicrosoft.com
-    - name: Log in with a service principal using client certificate.
+        az login --user johndoe@contoso.com --password secret -myPasswordThatStartsWithAdash
+    - name: Log in with a service principal using a client secret.
       text: >
-        az login --service-principal -u http://azure-cli-2016-08-05-14-31-15 -p ~/mycertfile.pem --tenant contoso.onmicrosoft.com
-    - name: Log in using a VM's system assigned identity
+        az login --service-principal --user http://azure-cli-2016-08-05-14-31-15 --password VerySecret --tenant contoso.onmicrosoft.com
+    - name: Log in with a service principal using a client certificate.
+      text: >
+        az login --service-principal --user http://azure-cli-2016-08-05-14-31-15 --password ~/mycertfile.pem --tenant contoso.onmicrosoft.com
+    - name: Log in using a VM's system assigned identity.
       text: >
         az login --identity
-    - name: Log in using a VM's user assigned identity. Client or object ids of the service identity also work
+    - name: Log in using a VM's user assigned identity. Client or object ids of the service identity also work.
       text: >
-        az login --identity -u /subscriptions/<subscriptionId>/resourcegroups/myRG/providers/Microsoft.ManagedIdentity/userAssignedIdentities/myID
+        az login --identity --user /subscriptions/<subscriptionId>/resourcegroups/myRG/providers/Microsoft.ManagedIdentity/userAssignedIdentities/myID
 """
 
 helps['account'] = """
 type: group
 short-summary: Manage Azure subscription information.
+long-summary: >
+  For an in-depth overview of working with subscriptions using the Azure CLI, see
+  `https://learn.microsoft.com/en-us/cli/azure/manage-azure-subscriptions-azure-cli`
```

Comment:
We should leave hyperlinks as is without `` ` ``, so that it is clickable in the online help.